amazon-sqs/5.5/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc

```diff
@@ -1,6 +1,4 @@
-= Amazon SQS Additional Configuration Information - Mule 4
-:page-aliases: connectors::amazon/amazon-sqs-connector-config-topics.adoc
-
+= Amazon SQS Connector 5.5 Additional Configuration Information - Mule 4
```

Comment:
Please mark 5.5.0 version as deprecated. Every user that arrived here should know that this version is deprecated. (Same for the remaining 5.5.0 docs)

Reply:
This is the old topic. The new one is in the 5.6 directory in this PR